solutions/filteredObj.js

```diff
@@ -0,0 +1,8 @@
+const filtObj=(obj,key,value)=>{
+  const func = (key,value,obj)=>{
```

Comment:
why are you writing another function within a function? it doesn't seem like you're doing anything different. just take the extra inner function out

Reply:
Also, where are you checking for the length of each value?? the point of filteredObj is to filter out the object but checking if each value (which is always a string)'s length is greater than 3.